usaspending_api/etl/management/commands/load_table_to_delta.py

```diff
@@ -83,7 +83,23 @@
         "custom_schema": "award_id LONG",
         "column_names": list(FINANCIAL_ACCOUNTS_BY_AWARDS_COLUMNS),
     },
-    "recipient_profile": {
+    "recipient_lookup": {
+        "model": RecipientLookup,
+        "is_from_broker": False,
+        "source_table": "recipient_lookup",
+        "source_database": "rpt",
+        "destination_database": "raw",
+        "swap_table": None,
+        "swap_schema": None,
+        "partition_column": "id",
+        "partition_column_type": "numeric",
+        "is_partition_column_unique": True,
+        "delta_table_create_sql": recipient_lookup_create_sql_string,
+        "source_schema": None,
+        "custom_schema": "recipient_hash STRING",
+        "column_names": list(RECIPIENT_LOOKUP_COLUMNS),
+    },
+    "recipient_profile_testing": {
```

Comment:
Think that was added in via the merge conflict handling. Also, can you move `recipient_profile_testing` to be grouped with the others under the comment `# Additional definitions for use in testing;`?